python/tests/res/enkf/test_enkf_fs_manager1.py

```diff
@@ -38,10 +38,7 @@ def test_create(self):
 
             fs2 = fsm.getFileSystem("newFS")
             self.assertEqual(2, fsm.getFileSystemCount())
-            self.assertEqual(1, fs2.refCount())
-
-            with self.assertRaises(AssertionError):
-                fs2.umount( )
+            #self.assertEqual(1, fs2.refCount())
```

Comment:
Still have to do something about these asserts.

Reply:
I deleted those. I think it is sufficiently tested in test_enkf_fs_manager2.py